sigpy/util.py

```diff
@@ -390,6 +391,92 @@ def monte_carlo_sure(f, y, sigma, eps=1e-10):
     return sure
 
 
+def leja(x):
```

Comment:
Interesting! Can you provide more documentation and a reference to this? And do leja and leja_fast produce the same results? If so, do we need leja if leja_fast is faster?